judge/views/contests.py

```diff
@@ -510,9 +510,12 @@ def get_context_data(self, **kwargs):
             queryset.values('problem__code', 'result').annotate(count=Count('result'))
                     .values_list('problem__code', 'result', 'count'),
         )
-        labels, codes = zip(
-            *self.object.contest_problems.order_by('order').values_list('problem__name', 'problem__code'),
-        )
+        labels, codes = [], []
+        contest_problems = self.object.contest_problems.order_by('order').values_list('problem__name', 'problem__code')
+        if contest_problems.exists():
```

Comment:
Why are you using `.exists()`? You are fetching it anyways so might as well hit the database once...

```suggestion
        if contest_problems:
```